packages/editor/src/components/block-icon/index.js

```diff
@@ -18,13 +28,12 @@ function renderIcon( icon ) {
 		}
 
 		return icon();
-	} else if ( icon && icon.type === 'svg' ) {
+	} else if ( isSVGIcon( icon ) ) {
```

Comment:
I wanted to write this way simpler, like this:

```
	} else if ( icon && icon.type === 'svg' || icon && icon.type.name === 'SVG' ) {
		const appliedProps = {
			...icon.props,
			width: icon.props.width || 24,
			height: icon.props.height || 24,
		};
		return <SVG { ...appliedProps } />;
	}
```

But for whatever reason, this did not pass linting tests. I'm not sure why, as this implementation seems far more complex.